consensus/src/block_storage/block_store.rs

```diff
@@ -478,26 +476,4 @@ impl BlockStore {
         self.insert_single_quorum_cert(block.quorum_cert().clone())?;
         Ok(self.execute_and_insert_block(block)?)
     }
-
-    /// Helper function to insert a reconfiguration block
-    pub fn insert_reconfiguration_block(&self, block: Block) -> anyhow::Result<Arc<ExecutedBlock>> {
-        self.insert_single_quorum_cert(block.quorum_cert().clone())?;
-        let executed_block = self.execute_block(block)?;
-        let compute_result = executed_block.compute_result();
-        let result = StateComputeResult::new(
-            compute_result.root_hash(),
-            compute_result.frozen_subtree_roots().clone(),
-            compute_result.num_leaves(),
-            compute_result.parent_frozen_subtree_roots().clone(),
-            compute_result.parent_num_leaves(),
-            Some(EpochState::empty()),
-            compute_result.compute_status().clone(),
-            compute_result.transaction_info_hashes().clone(),
-        );
-        Ok(self
-            .inner
-            .write()
-            .unwrap()
-            .insert_block(ExecutedBlock::new(executed_block.block().clone(), result))?)
-    }
```

Comment:
I suspect a rebase would eliminate those changes to consensus?

Reply:
No, that code is unused but because it was tied up in the mess of features, the linter never saw it. Now that the dependencies are more correctly expressed, I fixed the resulting linting errors.